test/e2e/01-install.sh

```diff
@@ -66,13 +66,17 @@ set -e
 kubectl create secret tls -n tekton-pipelines tekton-results-tls --cert="${SSL_CERT_PATH}/tekton-results-cert.pem" --key="${SSL_CERT_PATH}/tekton-results-key.pem" || true
 
 echo "Installing Tekton Results..."
-kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply -f -
+if [[ "$(uname -a)" == *ARM64* && "$(uname -m)" == x86_64* ]]; then
+    kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply --platform=linux/arm64 -f -
```

Comment:
Do we need to change this directory to `${ROOT}/config` ?

Reply:
`${ROOT}/config` is included https://github.com/tektoncd/results/blob/main/test/e2e/kustomize/kustomization.yaml#L16 . Does that answer your question?